pkg/log/scope.go

```diff
@@ -120,6 +120,24 @@ func (s *Scope) ErrorEnabled() bool {
 	return s.Level >= ErrorLevel
 }
 
+// Fatal outputs a message at error level, and then os.Exit.
```

Comment:
pkg/log has 100% code coverage, you need to add coverage for this new functionality. This will be tricky since it's calling os.Exit.

I suggest defining a global function pointer that is initialized by default to call os.Exit and that's what all this code calls through. In the test, you can replace this function pointer by something else.